server/cpho/views/view_util.py

```diff
@@ -231,6 +231,76 @@ def export_mapper():
     return export_mapping
 
 
+def export_mapper():
```

Comment:
something is wrong here:
There's two export mappers()

probably sth went wrong while resolving a merge conf